app/src/main/java/org/simple/clinic/drugs/PrescriptionRepository.kt

```diff
@@ -157,4 +157,22 @@ class PrescriptionRepository @Inject constructor(
         syncStatus = SyncStatus.PENDING
     )
   }
+
+  fun updateTeleconsultationId(drugUuid: UUID, teleconsultationId: UUID) {
+    dao.updateTeleconsultationId(
+        id = drugUuid,
+        teleconsultationId = teleconsultationId,
+        updatedAt = Instant.now(utcClock),
+        syncStatus = SyncStatus.PENDING
+    )
+  }
+
+  fun updateTeleconsultationIds(drugUuids: List<UUID>, teleconsultationId: UUID) {
+    dao.updateTeleconsultationIds(
+        drugUuids = drugUuids,
+        teleconsultationId = teleconsultationId,
+        updatedAt = Instant.now(utcClock),
+        syncStatus = SyncStatus.PENDING
+    )
+  }
```

Comment:
Can we rename this to better reflect what it is doing? Ex: `addTeleconsultationIdToDrugs`.